src/Inquirer/Prompts/IScreenManager.cs

```diff
@@ -7,6 +8,7 @@ public interface IScreenManager
     {
         int[,] RenderMultipleMessages(string[] messages);
         int[] Render(string[] content, string[] bottomContent);
+        int[] RenderList(ListInputMessage[] content, string[] bottomContent);
```

Comment:
I prefer to not have a specific render to list, so it can be just a Render.

Reply:
Great! I will try to implement only with the Render method instead of having an exclusive for ListInput.